README.Rmd

```diff
@@ -40,8 +40,104 @@ You can install the development version of elicitr from GitHub with:
 pak::pak("CREWdecisions/elicitr")
 ```
 
+### Getting started
+
+```{r}
+library(elicitr)
+```
+
+All the functions in the elicitr package start with the prefix `elic_`. After that, two prefixes are available: `elic_cont` and `elic_cat`. This design choice is intended to enhance functions discovery.\
+`elic_cont` functions are used for the elicitation of continuous variables and `elic_cat` functions for the elicitation of categorical variables.
+
+### Elicitation of continuous variables
+
+#### Simulated datasets
+
+Two simulated datasets are included in elicitr. These datasets are intended to demonstrate the functionality of the package and do not represent an actual elicitation process (names are also randomly generated).
+
+```{r}
+round_1
+```
+###
+```{r}
+round_2
+```
+
+#### How elicitr works
+
+For the analysis of continuous variables, the core of the elicitr package is an object of class `elic_cont`. Just as you create a form to collect estimates in an elicitation process, you need to create the `elic_cont` object to store metadata information. This allows to check whether all experts have given their answers in the expected way.\
+Any analysis starts by creating the `elic_cont` object with the function `elic_cont_start()`.To build this `elic_cont` object, Four parameters must be specified: 
+
+* `var` the number of variables (i.e. the number of topics in your elicitation)
+* `var_types` the type of variables for each of these variables (many options are available, ranging from real numbers to probabilities)
+* `elic_types` the type of elicitation for each of these variables (three options are available: one, three, and four points elicitations)
+* `experts` the number of experts that replied to the elicitation
+
+```{r}
+my_elicitation <- elic_cont_start(var = c("var1", "var2", "var3"),
+                                  var_types = "ZNp",
+                                  elic_types = "134",
+                                  experts = 6,
+                                  title = "Elicitation example")
+```
+###
+```{r, echo = FALSE}
+my_elicitation
+```
+
+Once the metadata has been added to the `elic_cont`object, the data of the first round of elicitation can be added with the function `elic_cont_add_data()`;
```

Comment:
Line 88: missing space between `elic_cont` and object. And sorry, I meant a colon not a semicolon at the end of the sentence 😅